try {
    log(a, b, c);
} catch (error) {
    console.log(error);
}

try {
} catch {
}

try {
    a();
    b();
} catch {
}

